### DISCUSSION TITLE
Where can I install player-made blueprints on the server?

### DISCUSSION THREAD

**Discussion by lifewater** [wolveix/satisfactory-server#180] [Q&A] [answered]
The documentation says blueprints are saved in /saved, but that folder doesn't exist for me, only /saves, so i just assumed it was a typo.

My /config volume is mounted from ~/satisfactory-server,  I've tried putting blueprints in:
`~/satisfactory-server/saves/
~/satisfactory-server/saves/blueprints/
~/satisfactory-server/saves/blueprints/Genesis <-- our session name` 

Also I'm unsure if its necessary to restart the docker when you place these blueprint files.  Any help is appreciated!

**Comment by wolveix** [comment, marked as answer]
Hey! If you see `/saves` instead of `/saved`, it's likely that you're using a much older image. Please pull the latest image and check what you see! Make sure you backup your `/saves` directory first, just in case

**Reply by lifewater** [reply]
Cancel, for whatever reason pulling :latest doesnt do what i thought it would. i had to manually force remove the image for it to work like youve mentioned

**Reply by lifewater** [reply]
Thanks this worked, blueprints are now loading after a container restart :)
